external_queue.go

```
@@ -74,23 +85,23 @@ func (wbq *externalQueue[T, R]) GroupsJobById(id string) (EnqueuedSingleGroupJob
 	return val.(EnqueuedSingleGroupJob[R]), nil
 }
 
-func (wbq *externalQueue[T, R]) WaitUntilFinished() {
+func (eq *externalQueue[T, R]) WaitUntilFinished() {
 	// to ignore deadlock error if the queue is paused
-	if wbq.IsPaused() {
-		wbq.Resume()
+	if eq.IsPaused() {
+		eq.Resume()
```

golangci-lint (1.64.8)

Error return value of `eq.Resume` is not checked

(errcheck)